rules/CodeQuality/Rector/Property/MakeEntityDateTimePropertyDateTimeInterfaceRector.php

```diff
@@ -25,6 +25,8 @@
  * @see related to maker bundle https://symfony.com/doc/current/bundles/SymfonyMakerBundle/index.html
  *
  * @see \Rector\Doctrine\Tests\CodeQuality\Rector\Property\MakeEntityDateTimePropertyDateTimeInterfaceRector\MakeEntityDateTimePropertyDateTimeInterfaceRectorTest
+ *
+ * @deprecated see https://github.com/rectorphp/rector-doctrine/issues/356 for reasoning
  */
 final class MakeEntityDateTimePropertyDateTimeInterfaceRector extends AbstractRector
```

Comment:
It should also implement `DeprecatedInterface`

Reply:
updated to implements `DeprecatedInterface` https://github.com/rectorphp/rector-doctrine/pull/358/commits/53c349eee30825cfc83ff892059fb12d33df649f